config/locales/defaults/en.yml

```diff
@@ -86,17 +86,17 @@ en:
       about: >
         Lots of words on a page can make content difficult for users to read online. The longer the
         page is, the less usable it is likely to be. You could investigate whether this content could
-        be shorter. Otherwise, does it have headings and bulletpoint to help users scan?
+        be shorter. Otherwise, does it have headings and bulletpoints to help users scan?
     pdf_count:
       title: 'Number of PDFs'
       short_title: 'Number of PDFs'
-      summary: 'Number of .pdf attachements on the page'
+      summary: 'Number of .pdf attachments on the page'
       context: ''
       unit: ''
       data_source: none
       about: >
         Compared with HTML content, information published in a PDF is harder to find, use and 
         maintain. PDFs can often be bad for accessibility and rarely comply with open standards.
         Serveral PDFs on one page may indicate that too many users needs are being met by them.
-        Read more about why to avoid PDFs.
+        <A href="https://gds.blog.gov.uk/2018/07/16/why-gov-uk-content-should-be-published-in-html-and-not-pdf/">Read more about why to avoid PDFs</A>.
```

Comment:
```suggestion
        <a href="https://gds.blog.gov.uk/2018/07/16/why-gov-uk-content-should-be-published-in-html-and-not-pdf/">Read more about why to avoid PDFs</a>.
```